configs/templates/unified-dev/spack.yaml

```diff
@@ -24,14 +24,18 @@ spack:
     - neptune-env           ^esmf@=8.8.0
     - neptune-python-env    ^esmf@=8.8.0
     - soca-env
-    - ufs-srw-app-env       ^esmf@=8.6.1 ^crtm@=3.1.1-build1
-    - ufs-weather-model-env ^esmf@=8.6.1 ^crtm@=3.1.1-build1
+    - ufs-srw-app-env       ^esmf@=8.8.0 ^crtm@=3.1.1-build1
+    - ufs-weather-model-env ^esmf@=8.8.0 ^crtm@=3.1.1-build1
```

Comment:
Should we really make this change? This affects every platform, and we have to go back and reinstall where we already started. Since the UFS loads esmf and mapl versions explicitly, and since both esmf@8.6.1+mapl@2.53.0 and esmf@8.8.0+mapl@2.53.0 are available (is this true, or only if the lines 35-38 are added below?), there is no harm keeping it as is?

We should make this change in develop, though, and we should also make it for the skylab-dev template if we do this for unified-dev.

Reply:
These two lines (27,28) are not necessary, since I don't know if anybody is using it anymore.
But lines 35-38 are for sure necessary.
I already removed same PR #1530  from develop, since this will be propagated to develop eventually, right?

Reply:
Ok. Sigh, we'll have to tell folks to stop again rolling out 1.9.1.